packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -325,7 +325,10 @@ class ConnectionStateHandler implements IConnectionStateHandler {
             } else if (this.shouldWaitForJoinSignal()) {
                 // timer has already fired, meaning it took too long to get join op/signal.
                 // Record how long it actually took to recover.
-                this.handler.logConnectionIssue("ReceivedJoinOp");
+                // This is generic event, as it by itself is not an error.
+                // We also have a case where NoJoinOp happens during container boot (we do not report it as error in such case),
+                // this this log statement happens after boot - we do not want to consider it error case.
```

Comment:
```suggestion
                // if this log statement happens after boot - we do not want to consider it error case.
```
I _think_ this is what was intended.